ADVANCED.md

```diff
@@ -408,7 +408,11 @@ import { Results } from "@elastic/react-search-ui";
 
 Shows a dropdown for selecting the number of results to show per page.
 
-Uses 20, 40, 60 as options.
+Uses [20, 40, 60] as a default options. You can use `resultsPerPageOptions` prop to pass custom options. 
+
+**Note:** When passing custom options make sure one of the option value matches 
+the current [`resultsPerPage`](#resultsPerPageProp) value, which is 20 by default.
+To override `resultsPerPage` default value [Refer the custom options example](#Example-using-custom-options).
```

Comment:
Small typos:

`Refer the custom options example` -> `refer to the custom options example`

`the option value matches` -> `the option values match`

Reply:
Sorry about that.